readme.md

```diff
@@ -33,4 +33,5 @@ Create a new dashboard and replace the `panels` value with this in the Grafana J
 https://grafana.com/docs/agent/latest/getting-started/create-config-file/
 
 
-
+## Exported metrics:
+A list of exported metrics can be found [here](./metrics.md)
```

Comment:
No new line at the end?

![](https://media.giphy.com/media/F3BeiZNq6VbDwyxzxF/giphy-downsized.gif)

Reply:
![frustrated-headache-gif-by-kelly-clarkson](https://github.com/supabase/grafana-agent-fly-example/assets/30039543/ec76c955-f57f-44a8-883b-bba9fca79aa2)